pkg/operator/operator.go

```diff
@@ -39,6 +41,11 @@ func New(metricsAddr string) (*Operator, error) {
 		return nil, fmt.Errorf("unable to register the thanos querier controller with the manager: %w", err)
 	}
 
+	ok := func(_ *http.Request) error { return nil }
+	if err := mgr.AddHealthzCheck("health probe", ok); err != nil {
```

Comment:
I guess we want to add some `ready` check as well. I am wondering what the difference would be.....

Reply:
for now, the service is always (healthy/ready) or (not healthy/not ready) so I don't think we need to differentiate and implementing only the health handler is fine.

Reply:
OK thanks.